### PR TITLE
Created md file containing link to asset

### PR DESCRIPTION
**Pull request by gianniconte**
Issuing my first pull request

### CODE REVIEW

**Comment by gianniconte** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by AlexanderHodicke — CHANGES_REQUESTED** [review]
I assume this is just a link change on a product page. No need for a new asset folder. Please change / or write me if I misunderstood something.

**Comment by gianniconte** [comment]
Hi Alexander.

It is not an asset. It is a link to a video demo.

It was first attempt, so I will do better accordingly.

Gianni

[cid:ce47353a-44c1-4da5-bb4f-6307c7468cf3]
Giovanni Conte
Specialist - Integration
[cid:f3619dc8-6e62-4cc4-a261-094db4872c1e]
Oracle Certified Professional
- Oracle Cloud Infrastructure (OCI)
- Cloud Platform Application Integration
Mobile: +39 348 1546002
Oracle Italy

[cid:4bf237d2-6259-4aa4-b951-10788cbaac19]

Oracle is committed to developing practices and products that help protect the environment
________________________________
Da: Alexander Hödicke ***@***.***>
Inviato: mercoledì 28 giugno 2023 11:41
A: oracle-devrel/technology-engineering ***@***.***>
Cc: Giovanni Conte ***@***.***>; Author ***@***.***>
Oggetto: [External] : Re: [oracle-devrel/technology-engineering] Created md file containing link to asset (PR #20)


@AlexanderHodicke requested changes on this pull request.

I assume this is just a link change on a product page. No need for a new asset folder. Please change / or write me if I misunderstood something.

File: application-development/application-integration-and-process-automation/oracle-integration-cloud/OIC - Recipes and Accelerators - Exploit Reuse/files/README.md
Comment (MarcelStraka) [inline application-development/application-integration-and-process-automation/oracle-integration-cloud/OIC - Recipes and Accelerators - Exploit Reuse/files/README.md:1]: Why do we have this folder?

**Comment by AlexanderHodicke** [comment]
PII information is found in the PR comment (business phone number) I have to reject and delete the branch. Please create a new branch and PR. Please use your web browser to reply through emails.

**Pull request closed without merging**